types/microsoft-ajax/index.d.ts

```diff
@@ -627,7 +627,7 @@ declare namespace Sys {
 
         //#region Constructors
 
-        constructor(): void;
+        new(): void;
```

Comment:
`void` is probably wrong, but it wasn't constructable before now sooo... 🤷‍♂ @pjmagee can you advise?

Reply:
Yes, `void` is wrong. 

But to have something more precise than `void` requires having some interface/class describing the instance. And just as with the `ProfileService` class, the constructor in the `Application` class is basically useless.    
And changing `Application` from an `interface` to a `class` causes other type errors.  

So keeping the `void` return-type seemed like the least bad option.  
(An alternative is to make it return `{}`).

Reply:
Right, I mean, if someone tries to use this, they’re _going_ to have problems. You can’t do anything to `void`. But, this interface literally wasn’t constructable before now, so it seems like nobody is using it or nobody cares. It’s very strange.